travis/jekyll_build.sh

```diff
@@ -29,7 +29,7 @@ kill %1
 bundle exec htmlproofer --check-img-http --check-html \
 --internal-domains 'forsetisecurity.org' \
 --check-favicon --report-missing-names --report-script-embeds \
---url-ignore '/GoogleCloudPlatform/forseti-security/edit/,/forseti-security/forseti-security/edit/,/maxcdn.bootstrapcdn.com/,/d3js.org/' \
+--url-ignore '/forseti-security/forseti-security/edit/,/forseti-security/forseti-security/edit/,/maxcdn.bootstrapcdn.com/,/d3js.org/' \
```

Comment:
I don't think this reference needed to be updated. We can either should keep it or if everything is renamed we can just remove it.

Reply:
Agree, it can just be removed since the next URL to be ignored is the forseti-security URL.

Reply:
Removed as everything else is updated.

Reply:
Do we will still need the other url ignores? I thought @dekuhn was just referencing the first url in that list.